app/javascript/dashboard/assets/scss/_variables.scss

```diff
@@ -67,6 +67,14 @@ $masked-bg: rgba(0, 0, 0, .4);
 $color-primary-light: #c7e3ff;
 $color-primary-dark: darken($color-woot, 20%);
 
+// Social and inboxes brand colors
+$facebook-brand-color: #3b5998;
+$twitter-brand-color: #1da1f2;
+$whatsapp-brand-color: #25d366;
+$sms-twilio-color: #f42f46;
+$chat-company-brand-color: $primary-color;
```

Comment:
We have `$color-woot` and `--color-woot`, So this is not needed.

Reply:
I added it for readability so that someone trying to figure out why ion-earth is their brand color sees that it's a reflection of color woot.